Dockerfile_pg

```diff
@@ -1,5 +1,5 @@
 # start with a base image
-FROM postgres:11
+FROM postgres:13.2
```

Comment:
Now that we're finally doing this, I don't think we need the `Dockerfile_pg_dev` file anymore ... think you can delete it and update `docker-compose.dev.yml`